tests/integration/test_model.py

```
@@ -0,0 +1,38 @@
+from pathlib import Path
+import pytest
+
+from dm_control import mjcf
+from stac_mjx.operations import mjx_load
+
+from pathlib import Path
```

Ruff

Redefinition of unused `Path` from line 1

Remove definition: `Path`

(F811)